v19/src/app/app.component.html

```
@@ -0,0 +1,24 @@
+/* 
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)